scripts/actions/send-and-update-translation-queue.js

```diff
@@ -172,6 +180,15 @@ const sendContentToVendor = async (content, accessToken) => {
   const jobUids = jobsResponses.map((resp) => resp.translationJobUid);
   console.log(`[*] Successfully created jobs: ${jobUids.join(', ')}`);
 
+  const queueTranslationsJobsRecord = jobUids.map((jobUid) =>
+    addTranslationsJobsRecord(jobUid)
```

Comment:
i think this needs translationId as well.

Reply:
oh, which id does that correlate to then, is it the translation id from the Translations table? I don't see how to correlate to that with the data we have at this point, i may need to rewrite this a bit but we should campfire 😬